jobsdb/backup.go

```diff
@@ -137,15 +138,30 @@ func (jd *Handle) uploadDumps(ctx context.Context, dumps map[string]string) erro
 		path := filePath
 		operation := func() error {
 			if err := jd.uploadTableDump(ctx, wrkId, path); err != nil {
-				jd.logger.Errorf("[JobsDB] :: Failed to upload workspaceId %v. Error: %s", wrkId, err.Error())
-				stats.Default.NewTaggedStat("backup_ds_failed", stats.CountType, stats.Tags{"customVal": jd.tablePrefix, "workspaceId": wrkId}).Increment()
+				jd.logger.Errorw(
+					"failed backup upload",
```

Comment:
```suggestion
					"failed to upload backup",
```

Reply:
going with "failed to upload jobs". The same is being used in archival.